compat/win32/pthread.c

```diff
@@ -57,6 +57,8 @@ pthread_t pthread_self(void)
 	return t;
 }
 
+#ifdef GIT_WIN_XP_SUPPORT
```

Comment:
@rongjiecomputer thanks for the ping. And yes, I absolutely agree: there may be interest by some XP users in trying to make other people work, but I have not heard of *anybody* who was even remotely interested in actually doing anything about resurrecting XP support. So that's a really clear indicator that we should now drop XP support for good.

I hate to put more work on your shoulders, but since you offered so nicely: yes, I would *love* to merge an updated PR that simply removes the XP-specific code instead of guarding it.